### PR TITLE
Origin/editorialreview/operate

### PR DESCRIPTION
**Pull request by christinaausley**
Editorial review of all Operate docs for grammar, punctuation, and terminology. Please let me know if you have any questions/comments!

### CODE REVIEW

File: docs/product-manuals/operate/deployment/authentication.md [hunk 1]
```diff
@@ -4,59 +4,60 @@ title: Authentication
...
   password: aPassword
 ```
 
-On Operate startup the user will be created if not existed before.
+On startup of Operate, the user is created if they did not exist before.
+
+By default, one user with **username**/**password** `demo`/`demo` is created.
 
-By default one user with **username**/**password** `demo`/`demo` will be created.
+Add more users directly to Elasticsearch via the index `operate-user-<version>_`. The password must be encoded with a strong bcrypt hashing function.
```
Comment (akeller) [inline docs/product-manuals/operate/deployment/authentication.md:31]: Nice catch here! For bcrypt, what do you think about handling it like `zbctl` where we wrap it in backticks so it renders as code?

File: docs/product-manuals/operate/deployment/importer-and-archiver.md [hunk 2]
```diff
@@ -72,13 +74,14 @@ camunda.operate:
   
 ```
 
-You can further parallelize archiver and(or) importer within one node by using following configuration parameters:
+You can further parallelize archiver and/or importer within one node using the following configuration parameters:
```
Comment (akeller) [inline docs/product-manuals/operate/deployment/importer-and-archiver.md:77]: Uses of and/or should be added to the style guide.

**Review by akeller — APPROVED** [review]
Looks good! Approving this since my comments are more conversational items than fixes 😄

File: docs/product-manuals/operate/deployment/configuration.md [hunk 3]
```diff
@@ -85,10 +87,13 @@ camunda.operate:
...
 ## Zeebe Elasticsearch exporter
 
-Operate imports data from Elasticsearch indices created and filled in by [Zeebe Elasticsearch Exporter](https://github.com/camunda-cloud/zeebe/tree/develop/exporters/elasticsearch-exporter).
-Therefore, settings for this Elasticsearch connection must be defined and must correspond to the settings on Zeebe side.
+Operate imports data from Elasticsearch indices created and filled in by the [Zeebe Elasticsearch Exporter](https://github.com/camunda-cloud/zeebe/tree/develop/exporters/elasticsearch-exporter).
+
+Therefore, settings for this Elasticsearch connection must be defined and must correspond to the settings on the Zeebe side.
 
-### Settings to connect and import:
+[//]:# (Do you mean instances in place of "indices" above?)
```
Comment (ralfpuchert) [inline docs/product-manuals/operate/deployment/configuration.md:94]: No, it's correct. Indices are imported.

File: docs/product-manuals/operate/deployment/configuration.md [hunk 4]
```diff
@@ -191,6 +197,8 @@ that can be further adjusted to your needs:
 
 By default, Console log appender will be used.
 
+[//]:# (Do you mean ConsoleAppender above?)
```
Comment (ralfpuchert) [inline docs/product-manuals/operate/deployment/configuration.md:200]: Yes, ConsoleAppender is meant.

**Review by ralfpuchert — DISMISSED** [review]
Thank you again!

File: docs/product-manuals/operate/deployment/authentication.md (hunk 1 above)
Comment (christinaausley) [reply docs/product-manuals/operate/deployment/authentication.md:31]: Agreed -- leaving it as ordinary text looks a little awkward. This is a good use for a code block. Will add into glossary!

File: docs/product-manuals/operate/deployment/importer-and-archiver.md (hunk 2 above)
Comment (christinaausley) [reply docs/product-manuals/operate/deployment/importer-and-archiver.md:77]: Added -- great note!

**Pull request merged**